### PR TITLE
adding an includes? method to ModelOpenStruct

### PR DESCRIPTION
**Pull request by Blackbaud-KyleMartinez**
@MatthewMazaika @Blackbaud-MarcusSimpson Adding includes? method to ModelOpenStruct to fix issue Aarron Hensley was having with with_updated_model

### CODE REVIEW

**Comment by Blackbaud-KyleMartinez** [comment]
@Blackbaud-AaronHensley

File: spec/open_sturct_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative 'spec_helper'
+
+describe "ModelOpenStruct includes?" do
```
Comment (MatthewMazaika) [inline spec/open_sturct_spec.rb]: let's add a case for a single element hash as well as a case where someone doesn't pass a hash

Comment (MatthewMazaika) [inline spec/open_sturct_spec.rb]: is there anything else that someone might pass that we want to catch?

**Comment by MatthewMazaika** [comment]
can we write a test that covers the issue that @Blackbaud-AaronHensley originally ran into using `with_updated_model` - or is that a CLO thing?

**Comment by Blackbaud-KyleMartinez** [comment]
@Blackbaud-AaronHensley @MatthewMazaika fixed typo and raising error is Hash is not a hash :)

**Comment by MatthewMazaika** [comment]
i agree that for this, `with_updated_model` wasn't the problem and this should fix it, but i'm saying we should throw some better tests around `with_updated_model` to make sure it's doing what we expect it to do in the case it gets a `ModelOpenStruct` instead of a regular `Model`

**Comment by Blackbaud-KyleMartinez** [comment]
After looking at this I'm not sure how it would be done. with_updated_model takes in a cucumber table.

**Pull request merged**